src/main/resources/i18n/en.json

```diff
@@ -119,5 +119,13 @@
     "calendar.event.add": "Add",
     "calendar.event.attachments" : "Attachments",
     "calendar.event.attachment.download.error" : "Error when trying to download attachment",
-    "calendar.event.timeslot": "Timeslot"
+    "calendar.event.timeslot": "Timeslot",
+
+    "calendar.rbs.sniplet.booking.owner.is": "Resources were booked by",
```

Comment:
i18n migrated from RBS because only used in calendar component (see https://github.com/OPEN-ENT-NG/rbs/pull/19)